HarpGuru/HarpFace/HarpCell/analysePosition/index.ts

```diff
@@ -10,7 +16,11 @@ export type PositionFacts = {
   readonly leftmost: boolean
 }
 
-export const analysePosition = (props: HarpCellProps): PositionFacts => {
+type Props = HarpCellProps & {
```

Comment:
In the end I don't think it will be helpful to have these helper functions even refer to their associated components type. This was much more appropriate when we were trying to save ourselves maintenance effort by just drilling props straight through the app, deliberately leaving redundant props in place. Now this is not required.

I may leave this and others just like this for now though since the purpose of this pull request is to be a light touch and many decisions will be made once reactn is bedded in which will likely cover off questions like this much more efficiently than dealing with them up front before we know where we are going.

If I don't fix this though, I should associate it to an issue so that it and everything like it are not overlooked.

Reply:
https://github.com/js-jslog/harpnative/issues/45 raised to make sure that this and other things like it across all the pull requests associated with introducing reactn are addressed in due course.